docs/content/sips/019-governance.md

```diff
@@ -0,0 +1,127 @@
+
+The default decision making process is objection-free consensus. In other words, a decision is made when all decision makers have had time to consider the decision and do not raise any objections. Silence on any consensus decision is equivalent to non-objection. Explicit agreement may be stated at will.
+
+Decision making scenarios should be promoted appropriately by the maintainer overseeing the issue:
+
+- tag maintainers in relevant issues
+- send out mailing list reminders
+- communicate updates in meetings
+
+When a consensus cannot be found, a maintainer can call for a [simple majority](https://en.wikipedia.org/wiki/Majority) vote from the project maintainers for a repo on a decision. If the decision impacts multiple repositories or requires a coordinated effort across multiple repositories and project maintainers are unable to reach a decision on their own for the relevant projects, a maintainer can call for a simple majority vote from the Spin Governance Committee. If quorum cannot be met for a decision, all members of the Spin Governance Committee are added to the relevant vote.
+
+## **Spin Governance Committee**
+
+The project maintainers for [github.com/fermyon/spin](http://github.com/fermyon/spin) also serve as the Spin Governance Committee and have the following additional responsibilities:
+
+- Maintaining the mission, vision, values, and scope of the project
+- Refining this governance document
+- Making project level decisions
+- Making decisions about project licensing, contribution requirements, and other legal matters
+- Resolving escalated project decisions when responsible project maintainers are blocked
+- Managing the Spin brand
+- Managing access to Spin assets such as source repositories, hosting, project calendars
+- Handling code of conduct violations for any repository under the Spin project
+- Deciding what sub-groups and repositories are part of the Spin project
+- Overseeing the resolution and disclosure of security issues
+- Managing financial decisions related to the project
+
+In addition to the responsibilities listed above, this group is also responsible for bootstrapping a multi-stakeholder steering committee of 5-7 people to govern the project. The group is responsible for creating a steering committee Charter and shaping and executing the processes around selecting committee members. Once a steering committee is in place, the Spin Governance Committee will be dismantled and replaced with the Spin Steering Committee. The Spin Steering Committee will be responsible for bootstrapping a Code of Conduct Committee.
+
+The Spin Governance Committee have to match the following criteria:
+
+- Spin Governance Committee MUST remain active on the project. If they are unresponsive for > 3 months, they will lose membership, unless a [supermajority](https://en.wikipedia.org/wiki/Supermajority#Two-thirds_vote) of the other governance members agrees to extend the period to be greater than 3 months
```

Comment:
I believe this is what was meant (right now it sort of reads like we're talking about the committee as a whole instead of requirements placed on the individual members). 

```suggestion
- Spin Governance Committee members MUST remain active on the project. If they are unresponsive for > 3 months, they will lose membership, unless a [supermajority](https://en.wikipedia.org/wiki/Supermajority#Two-thirds_vote) of the other governance members agrees to extend the period to be greater than 3 months
```